mvpa2/base/externals.py

```diff
@@ -730,7 +728,7 @@ def check_all_dependencies(force=False, verbosity=1):
                 warning("%s is not available." % dep)
 
     if __debug__:
-        debug('EXT', 'The following optional externals are present: %s' \
-                     % [ k[5:] for k in cfg.options('externals')
-                            if k.startswith('have') \
-                            and cfg.getboolean('externals', k) == True ])
+        debug('EXT', 'The following optional externals are present: %s'
+                     % [k[5:] for k in cfg.options('externals')
```

Comment:
PEP8: "The preferred place to break around a binary operator is after the operator, not before it. "